app/models/force/application.rb

```diff
@@ -17,7 +19,7 @@ class Application < Force::ObjectBase
       { custom_api: 'applicationSubmissionType', domain: 'application_submission_type', salesforce: 'Application_Submission_Type' },
       { custom_api: 'applicationSubmittedDate', domain: 'application_submitted_date', salesforce: 'Application_Submitted_Date' },
       { custom_api: 'appRTType', domain: '', salesforce: '' },
-      { custom_api: '', domain: 'created_by', salesforce: 'CreatedBy' },
+      { custom_api: '', domain: 'createdby', salesforce: 'CreatedBy' },
```

Comment:
whoops

Reply:
this is intentional - the React app expects "createdby". i'd like to change that eventually, but for now the approach in this PR is not to go into the React app and start changing field names

Reply:
oops, I didn't mean that it was a mistake that you made that change, more just a mistake that we are expecting createdby in react